prow/config/jobs/istio.yaml

```
@@ -250,6 +250,20 @@ jobs:
     - prow/config/ambient-sc.yaml
     - test.integration.ambient.kube
 
+  - name: integ-ambient-multicluster
+    requirements: [kind]
```

```suggestion
    modifiers: [presubmit_optional]
    requirements: [kind]
```

Also a thought, make this optional while the testing is added to make sure everything is working as expected before we make it required.

---

Completely forgot that was an option; thanks for the catch